docs/src/miden_library.md

```diff
@@ -0,0 +1,128 @@
+| `compute_delta_commitment` | Computes the commitment to the native account's delta. Can only be called from auth procedures.<br><br>Inputs: `[]`<br>Outputs: `[DELTA_COMMITMENT]`<br>Context: Auth |
+| `incr_nonce` | Increments the account nonce by one and returns the new nonce. Can only be called from auth procedures.<br><br>Inputs: `[]`<br>Outputs: `[final_nonce]`<br>Context: Auth |
+| `get_item` | Gets an item from the account storage.<br><br>Inputs: `[index]`<br>Outputs: `[VALUE]`<br>Context: Account |
+| `set_item` | Sets an item in the account storage.<br><br>Inputs: `[index, VALUE]`<br>Outputs: `[OLD_VALUE]`<br>Context: Native & Account |
+| `get_map_item` | Returns the VALUE located under the specified KEY within the map contained in the given account storage slot.<br><br>Inputs: `[index, KEY]`<br>Outputs: `[VALUE]`<br>Context: Account |
+| `set_map_item` | Sets VALUE under the specified KEY within the map contained in the given account storage slot.<br><br>Inputs: `[index, KEY, VALUE]`<br>Outputs: `[OLD_MAP_ROOT, OLD_MAP_VALUE]`<br>Context: Native & Account |
+| `get_code_commitment` | Gets the account code commitment of the current account.<br><br>Inputs: `[]`<br>Outputs: `[CODE_COMMITMENT]`<br>Context: Account |
+| `get_initial_storage_commitment` | Returns the storage commitment of the native account at the beginning of the transaction.<br><br>Inputs: `[]`<br>Outputs: `[INIT_STORAGE_COMMITMENT]`<br>Context: Any |
+| `compute_storage_commitment` | Computes the latest account storage commitment of the current account.<br><br>Inputs: `[]`<br>Outputs: `[STORAGE_COMMITMENT]`<br>Context: Account |
+| `get_balance` | Returns the balance of the fungible asset associated with the provided faucet_id in the current account's vault.<br><br>Inputs: `[faucet_id_prefix, faucet_id_suffix]`<br>Outputs: `[balance]`<br>Context: Any |
+| `has_non_fungible_asset` | Returns a boolean indicating whether the non-fungible asset is present in the current account's vault.<br><br>Inputs: `[ASSET]`<br>Outputs: `[has_asset]`<br>Context: Any |
+| `add_asset` | Adds the specified asset to the vault. For fungible assets, returns the total after addition.<br><br>Inputs: `[ASSET]`<br>Outputs: `[ASSET']`<br>Context: Native & Account |
+| `remove_asset` | Removes the specified asset from the vault.<br><br>Inputs: `[ASSET]`<br>Outputs: `[ASSET]`<br>Context: Native & Account |
+| `get_initial_vault_root` | Returns the vault root of the native account at the beginning of the transaction.<br><br>Inputs: `[]`<br>Outputs: `[INIT_VAULT_ROOT]`<br>Context: Any |
+| `get_vault_root` | Returns the vault root of the current account.<br><br>Inputs: `[]`<br>Outputs: `[VAULT_ROOT]`<br>Context: Any |
+| `was_procedure_called` | Checks if a procedure has been called during transaction execution.<br><br>Inputs: `[PROC_ROOT]`<br>Outputs: `[was_called]`<br>Context: Any |
```

Comment:
Maybe the description should be "Returns 1 if a procedure was called during transaction execution, and 0 otherwise."